### PR TITLE
[CU-1m3fe9p] Feature/add image input field 

### PR DESCRIPTION
**Pull request by faejr**

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-1m3fe9p UI Component: Image Upload with Preview](https://app.clickup.com/t/1m3fe9p)

File: source/sass/component/_imageinput.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.c-imageinput {
+  .c-imageinput__preview {
+    display: block;
+
+    .c-imageinput__image {
```
Comment (nRamstedt) [inline source/sass/component/_imageinput.scss:5]: .c-imageinput__preview and .c-imageinput__image should be seperated (no need to nest them) unless there is a reason why image is dependent on a preview parent

File: source/js/fields.js [hunk 2]
```diff
@@ -127,6 +130,22 @@ class Fields {
         }
     }
 
+    imageDrop(formInput) {
+        const imagePreviewId = formInput.parentNode.getAttribute('data-image-preview');
+        const imagePreviewArea = document.getElementById(imagePreviewId);
+        imagePreviewArea.addEventListener('dragover', (e) => {
```
Comment (nRamstedt) [inline source/js/fields.js:141]: does this work even when/if imagePreviewArea is undefined?

**Review by nRamstedt — CHANGES_REQUESTED** [review]

File: source/js/fields.js (hunk 2 above)
Comment (faejr) [reply source/js/fields.js:141]: That's a fair point, this method should never run unless `data-image-preview` is defined on the input field. In which case imagePreviewArea _should_ be defined but I suppose a fail safe would be good to add

**Review by nRamstedt — APPROVED** [review]

**Pull request merged**